packages/orbit-components/src/ErrorFormTooltip/__tests__/index.test.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import ErrorFormTooltip from "..";
+
+describe("ErrorFormTooltip", () => {
+  it("should have error", () => {
+    const onClose = jest.fn();
+
+    render(
+      <ErrorFormTooltip
+        onClose={onClose}
+        onShow={() => {}}
+        dataTest="test"
+        tooltipShown
+        error="error"
+      />,
+    );
+
+    userEvent.click((document.body: any));
```

Comment:
To make this more clear, you could surround `ErrorFormTooltip` with another element and click on that instead? `data-test` could be something informative about what you're doing, e.g. `data-test="background"`.